.github/CODEOWNERS

```diff
@@ -76,4 +76,17 @@
 /model/trace/gen-ai.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-llm-approvers
 /docs/gen-ai/ @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-llm-approvers
 
+# Security semantic conventions approvers
+/model/registry/file.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/dns.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
+/model/registry/network.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-security-approvers
```

Comment:
it's already defined on line 40 (and owners include semconv-http-approvers)

Reply:
fixed, thanks for noticing it